### DISCUSSION TITLE
Is example01 TextInput supposed to print entered characters onto the command-line?

### DISCUSSION THREAD

**Discussion by willbprog127** [osch/lua-lwtk#4] [Q&A] [answered]
Hi!

I'm new to Lua and also to lua-lwtk.  After installing lua-lwtk using the github-hosted rockspec I tried running example01.lua and, while the window shows up, anything I type into the TextInput does not display inside the TextInput, but instead prints onto the command-line where I ran `lua example01.lua`.  Is this expected behavior or is there something funky going on?

Thanks!

* macOS Monterey - Intel
* Lua 5.4.4
* Examples from lua-lwtk repo .zip download

**Comment by osch** [comment, marked as answer]
This is a special MacOS problem: For macOS GUI programs you need a special folder structure with a `Info.plist` file (also known as "bundle"). See for more details: https://github.com/osch/lua-lwtk/issues/1

**Reply by willbprog127** [reply]
Sorry, did not see that issue.  Will check it out...

Thanks! 👍

**Reply by osch** [reply]
No problem, this topic should better be mentioned somewhere in the documentation somehow...
